src/com/magento/idea/magento2plugin/linemarker/xml/LineMarkerXmlTagDecorator.java

```diff
@@ -464,32 +439,49 @@ public PsiElement add(@NotNull PsiElement psiElement) throws IncorrectOperationE
     }
 
     @Override
-    public PsiElement addBefore(@NotNull PsiElement psiElement, @Nullable PsiElement psiElement1) throws IncorrectOperationException {
+    public PsiElement addBefore(
+            @NotNull PsiElement psiElement,
+            @Nullable PsiElement psiElement1
+    ) throws IncorrectOperationException {
         return xmlTag.addBefore(psiElement, psiElement1);
     }
 
     @Override
-    public PsiElement addAfter(@NotNull PsiElement psiElement, @Nullable PsiElement psiElement1) throws IncorrectOperationException {
+    public PsiElement addAfter(
+            @NotNull PsiElement psiElement,
+            @Nullable PsiElement psiElement1
+    ) throws IncorrectOperationException {
         return xmlTag.addAfter(psiElement, psiElement1);
     }
 
     @Override
     public void checkAdd(@NotNull PsiElement psiElement) throws IncorrectOperationException {
```

Comment:
Are we using this method anywhere?

Reply:
Nope

Reply:
Is this method declared in the `com.intellij.psi.xml.XmlTag`?

Reply:
The method from interface PsiElement

Reply:
com.intellij.psi.xml.XmlTag extend PsiElement